cmsswdata.spec

```diff
@@ -66,15 +63,12 @@ Requires: data-RecoHGCal-TICL
 Requires: data-SimG4CMS-HGCalTestBeam
 Requires: data-SimPPS-PPSPixelDigiProducer
 
-%if %isnotonline
```

Comment:
@smuzaffar out of curiosity, what is the reason of this change? For sure it makes life simpler, as there is no more distinction between what is "online" and the rest, but I wonder why this was in place there earlier at this point

Reply:
in past we had architectures like `slc5onl_amd64_gcc*` for special online builds. We are not building any more for those archs, so its time to clean up.

Reply:
@smuzaffar ok, so no reason to make it more complex than it needs to be now